commands/team.go

```diff
@@ -176,17 +177,26 @@ func archiveTeamsCmdF(c client.Client, cmd *cobra.Command, args []string) error
 }
 
 func listTeamsCmdF(c client.Client, cmd *cobra.Command, args []string) error {
-	teams, response := c.GetAllTeams("", 0, 10000)
-	if response.Error != nil {
-		return response.Error
-	}
+	page := 0
+	for {
+		teams, response := c.GetAllTeams("", page, web.LIMIT_MAXIMUM)
```

Comment:
ok, importing this `web.LIMIT_MAXIMUM` constant is triggering here the inclusion in the vendor directory a zillion of new files. I think can be more interesting to have a copy of the constant and avoid all that vendors directory explosion.